backend/api/Services/DocumentService.cs

```diff
@@ -74,11 +74,10 @@ public async Task<bool> DeleteActivityDocumentAsync(PimsActivityInstanceDocument
             this.Logger.LogInformation("Deleting PIMS document for single activity");
             this.User.ThrowIfNotAuthorized(Permissions.DocumentDelete);
 
-            IList<PimsActivityInstanceDocument> existingActivityDocuments = documentActivityRespository.GetAllByDocument(activityDocument.DocumentId);
+            IList<PimsActivityInstanceDocument> existingActivityDocuments = documentActivityRespository.GetAllByDocument(activityDocument.DocumentId, true);
```

Comment:
nit: documentActivityRe(s)pository -> documentActivityRepository

Reply:
Also while not caused by this PR, it would be nice to remove duplicated assignment from constructor
![image](https://user-images.githubusercontent.com/24322224/190274442-7dc8cca5-d64d-428c-b06b-559539ea15ff.png)